api/src/config/environment/index.ts

```diff
@@ -32,6 +32,7 @@ const base = {
   STRIPE_API_KEY: process.env.STRIPE_API_KEY || '',
   STRIPE_WEBHOOK_SECRET: process.env.STRIPE_WEBHOOK_SECRET || '',
   MONTHLY_ACTIVE_USERS_LIMIT: process.env.MONTHLY_ACTIVE_USERS_LIMIT || 2000,
+  NEXT_PUBLIC_AMPLITUDE_API_KEY: process.env.NEXT_PUBLIC_AMPLITUDE_API_KEY || '',
```

Comment:
Should be some private key
Probably just naming misunderstanding, `NEXT_PUBLIC` prefix used for next app only